low-level/files/opt/seagate/sspl/setup/resource_map/storage.py

```diff
@@ -49,7 +49,8 @@ def __init__(self):
             "psus": self.get_psu_info,
             "platform_sensors": self.get_platform_sensors_info,
             "logical_volumes": self.get_logical_volumes_info,
```

Comment:
lets rename this map to storage_map{} instead of keeping just frus, in name atleast

Reply:
done